src/usr/local/www/system_advanced_network.php

```diff
@@ -186,6 +199,15 @@
 ))->setHelp('By default, if IPv6 is configured and a hostname resolves IPv6 and IPv4 addresses, '. 
 	'IPv6 will be used. If this option is selected, IPv4 will be preferred over IPv6.');
 
+$section->addInput(new Form_Input(
+	'global-v6duid',
+	'DHCP6 DUID',
+	'text',
+	$pconfig['global-v6duid'],
+	['placeholder' => 'xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx']
+	))->setWidth(9)->sethelp('Enter the DUID to use here. Use this option if using RAM Disk, as the DUID will be lost on reboot. The existing DUID may be found in var/db/dhcp6_duid.' .'<br />' .
+			'NOTE: Do not use this option with multiple DHCP6 WAN interfaces.');
```

Comment:
The RFC doesn't say that. None that I've seen anyhow. While the DUID can be generated using an interface address as part of the value, DUID is not intended to be related to any interface. It's a client identifier that uniquely identifies a _device_ and not an interface. Identifying specific interfaces is where IAIDs come in, which we've been over.